apps/src/dropletUtils.js

```diff
@@ -625,17 +625,55 @@ exports.getAllAvailableDropletBlocks = function (dropletConfig, codeFunctions, p
  * @throws {Error} encountered unexpected Droplet token
  */
 exports.getFirstParam = function (methodName, block, editor) {
+  return getParamAtIndex(0, methodName, block, editor);
+};
+
+/**
+ * Gets the second parameter of the given function name, given either that
+ * function's DropletBlock or an AceEditor instance with its current cursor
+ * set to after the method.
+ *
+ *
```

Comment:
Nit: extra blank line (I personally wouldn't bother with any blank lines between description and params, but one is fine if you prefer).